settings.py

```diff
@@ -5,6 +5,9 @@
 # Argument parsing
 parser = argparse.ArgumentParser(description="GPU argument paser")
```

Comment:
This is not only a GPU argument parser now right?

Reply:
That's right. Updating